src/pages/RequestInvite/index.tsx

```diff
@@ -60,19 +60,13 @@ interface OwnProps {
 type Props = OwnProps & {
   authFullName: string | null;
   postOperation: typeof postOperation;
-  fetchMemberByMidIfNeeded: typeof fetchMemberByMidIfNeeded;
-  fetchMemberByUidIfNeeded: typeof fetchMemberByUidIfNeeded;
-  isAuthLoaded: boolean;
-  isAuthMemberDocLoaded: boolean;
-  isToMemberDocLoaded: boolean;
-  memberId: string;
-  toMemberDoc: MemberDoc;
-  notSignedIn: boolean;
-  authFirebaseUser: { uid: string };
+  targetMember: Member;
+  authFirebaseUser: { uid: string } | null;
```

Comment:
Unfortunately we can't assume the existence of a Member object as created by a RequestInvite operation since this page is intended for people who haven't requested an invite yet.

Agreed however, that the API endpoint should just consume an auth token instead of having the requesting user be explicitly specified. Moving over to Raha-API should do this.